packages/ast-spec/package.json

```
@@ -32,22 +32,22 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "//": "These package scripts are mostly here for convenience. Task running is handled by Nx at the root level.",
-    "build": "yarn run -BT nx build",
+    "build": "tsc -b tsconfig.build.json && api-extractor run --local --config=./api-extractor.json",
     "clean": "rimraf dist/ coverage/",
     "clean-fixtures": "rimraf -g \"./src/**/fixtures/**/snapshots\"",
-    "format": "yarn run -T format",
-    "lint": "yarn run -BT nx lint",
-    "test": "yarn run -BT nx test",
-    "typecheck": "yarn run -BT nx typecheck"
+    "format": "pnpm --workspace-root run format",
+    "lint": "pnpm --workspace-root exec nx lint",
+    "test": "pnpm --workspace-root exec nx test",
+    "typecheck": "pnpm --workspace-root exec nx typecheck"
```

Can shorten them to:

```suggestion
    "format": "pnpm -w run format",
    "lint": "pnpm -w exec nx lint",
    "test": "pnpm -w exec nx test",
    "typecheck": "pnpm -w exec nx typecheck"
```